packages/features/eventtypes/components/tabs/webhooks/EventWebhooksTab.tsx

```
@@ -236,7 +237,8 @@
               <WebhookForm
                 noRoutingFormTriggers={true}
                 webhook={webhookToEdit}
-                apps={installedApps?.items.map((app) => app.slug)}
+                apps={[...installedApps?.items.map((app) => app.slug), "cal-video"]}
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)